mumblepahelper.ui

```diff
@@ -564,6 +717,16 @@
     <string>&amp;Rescan</string>
    </property>
   </action>
+  <action name="action_SetUserPluginsLocation">
+   <property name="text">
+    <string>&amp;Set user plugins location...</string>
+   </property>
+  </action>
+  <action name="action_SetSystemPluginsLocation">
+   <property name="text">
+    <string>&amp;Set system plugins location...</string>
```

Comment:
This and the user plugin action use the same shortcut (&S). Shouldn't they use distinct ones?